src/components/app/albumsongscontainer/album-songs-container.jsx

```diff
@@ -26,20 +26,20 @@ class AlbumSongsContainer extends Component {
   }
 
   handleClickOutside() {
-    const { songsStore } = this.props
+    const { albumStore } = this.props
 
     if (this.wrapperRef && !this.wrapperRef.contains(event.target)) {
-      songsStore.clearSongsStore()
+      albumStore.clearalbumStore()
```

Comment:
imho `albumStore.clear` wystarczy. Po co powtarzać 2 razy to samo :P

Reply:
w sumie fakt, done